eta/core/video.py

```diff
@@ -2211,7 +2245,9 @@ def extract_clip(
 
     Args:
         video_path: the path to the video
-        output_path: the path to write the extracted video clip
+        output_path: the path to write the extracted video clip. If
+            `fast == True`, this path can optionally be a pattern like
```

Comment:
It is good to know that readers of the documentation will know this is possible. I would advocate for throwing an error, though, as it is not the methods intended use.

With all that said, I am assuming this was not changed for backwards compatibility.